data/pc/1.6.4/protocol.json

```diff
@@ -0,0 +1,2286 @@
+              {
+                "compareTo": "mode",
+                "fields": {
+                  "0": "string",
+                  "2": "string"
+                },
+                "default": "void"
+              }
+            ]
+          },
+          {
+            "name": "friendlyFire",
+            "type": [
+              "switch",
+              {
+                "compareTo": "mode",
+                "fields": {
+                  "0": "i8",
+                  "2": "i8"
+                },
+                "default": "void"
+              }
+            ]
+          },
+          {
+            "name": "players",
+            "type": [
+              "switch",
+              {
+                "compareTo": "mode",
+                "fields": {
+                  "0": [
+                    "array",
+                    {
+                      "countType": "i16",
+                      "type": "string"
+                    }
+                  ],
+                  "3": [
+                    "array",
+                    {
+                      "countType": "i16",
+                      "type": "string"
+                    }
+                  ],
+                  "4": [
+                    "array",
+                    {
+                      "countType": "i16",
+                      "type": "string"
+                    }
+                  ]
+                },
+                "default": "void"
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_custom_payload": [
+        "container",
+        [
+          {
+            "name": "channel",
+            "type": "string"
+          },
+          {
+            "name": "data",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_encryption_key_response": [
+        "container",
+        [
+          {
+            "name": "sharedSecret",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          },
+          {
+            "name": "verifyToken",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_encryption_key_request": [
+        "container",
+        [
+          {
+            "name": "serverId",
+            "type": "string"
+          },
+          {
+            "name": "publicKey",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          },
+          {
+            "name": "verifyToken",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_legacy_server_list_ping": [
```

Comment:
This should just be packet_server_list_ping the server responds using a kick packet.

It has the following fields seen in source and at https://wiki.vg/Server_List_Ping#1.6

a byte that just always holds 1
a byte that for some reason holds the ID of the CustomPayloadPacket
a string - This is just a constanct in this case its always "MC|PingHost"
a short  Length of the rest of the data in the packet
a byte - protocol version
a string  - host name of the server
a int - port of the server